performancecopilot/pcp
# qa/1661 diags

Changes committed to git@github.com:kmcdonell/pcp.git 20200130

Ken McDonell (3):
      Revert "qa/1661: adding some diags to help diagnose failures."
      qa/1661: reinstate diagnostics
      qa/1661: second part of missing diagnostics

Mark Goodwin (1):
      qa: fix 1661 to work with multiple pmlogger/farm and add _check_series check

 qa/1661 |   72 +++++++++++++++++++++++++++++++++++-----------------------------
 1 file changed, 40 insertions(+), 32 deletions(-)

Details ...

commit c41eac2f86d9f05c65b19bce458b74a6fa7aa71b
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Thu Jan 30 17:24:15 2020 +1100

    qa/1661: second part of missing diagnostics

commit d6cd8a894a1a069c333b5642462a986c7b82cb6e
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Thu Jan 30 17:06:24 2020 +1100

    qa/1661: reinstate diagnostics
    
    Put commit b612f9c7 back.

commit 2623ad74ff07c394f1339888ddf5aeac908ec92d
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Thu Jan 30 16:10:30 2020 +1100

    Revert "qa/1661: adding some diags to help diagnose failures."
    
    This reverts commit b612f9c7f3d6c782afa90cb4b182496293ab45af.
    
    Moving my changes asside to avoid merge conflict with Mark's
    next change.

commit 99d96694b76b6fed915efdded3cc24e3a0f911d6
Author: Mark Goodwin <mgoodwin@redhat.com>
Date:   Thu Jan 30 15:09:18 2020 +1100

    qa: fix 1661 to work with multiple pmlogger/farm and add _check_series check
    
    Add _check_series check to the test which will _notrun on
    platforms that do not support pmseries, e.g. no libuv.
    This might fix Ken's issues with this test on opensuse,
    
    Also fix the curvol check to work on systems with multiple
    pmloggers running - just check the current logvol for one
    of the pmloggers. This should fix Nathan's issues with this
    test.
